compute/reservations/sharedReservationConsumerProjectsUpdate.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-dupe-keys */
```

Comment:
Reason for: /* eslint-disable no-dupe-keys */ 
According to the documentation: https://cloud.google.com/compute/docs/instances/reservations-modify#modifying-consumer-projects-shared-reservation. When we want to update consumer projects, we must define paths field for each consumer, e.g:
`await reservationsClient.update({
      project: projectId,
      reservation: reservationName,
      paths: 'shareSettings.projectMap.consumerId1',
      paths: 'shareSettings.projectMap.consumerId2'
      zone,
      reservationResource: {
        name: reservationName,
        shareSettings: {
          projectMap: {
            consumerId1: {
              projectId: 'consumerId1',
            },
            consumerId2: {
              projectId: 'consumerId2',
            },
          },
        },
      },
    });`